docs/sidebars.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
     {
       '✨ Apps': [
         {
-          UI: ['apps/ui/introduction', 'apps/ui/ui-quick-start', 'apps/ui/inbox', 'apps/ui/tags', 'apps/ui/components'],
+          UI: ['apps/ui/introduction', 'apps/ui/ui-quickstart', 'apps/ui/inbox', 'apps/ui/tags', 'apps/ui/components'],
```

Comment:
since we're touching this... can we rename this file and drop the `ui-` it seems really redundant